experimental/python-flask/README.md

```diff
@@ -0,0 +1,66 @@
+# Python Flask Stack
+
+The Python Flask stack provides a consistent way of developing web applications using [Flask](http://flask.pocoo.org). "Flask is a lightweight WSGI web application framework. It is designed to make getting started quick and easy, with the ability to scale up to complex applications."
+
+This stack is based on Python 3.7 and Flask 0.11.1 and enables health checking and application metrics out of the box. The stack also provides a set of unit tests. The stack also uses [flasgger](https://github.com/rochacbruno-archive/flasgger) to auto-generate swagger ui
+documentation and specification.
+
+Remote debugging of applications is enabled during `debug` mode using `ptvsd`, which enables you to connect using the debugger in VS Code. A VS Code debug launch configuration is also provided.
+
+Note that the use of the Python Flash stack requires that `pipenv` is installed, which is uses for both version management of Python versions (ensuring you are using the same version locally as used in the stack), and to allow you to specify your own dependencies in a Pipfile.
```

Comment:
A few typos:
Python Flash -> Python Flask 
which is uses -> which is used

Reply:
coding is fine....its English I struggle with....

Reply:
done